conda/install.py

```diff
@@ -232,16 +232,18 @@ def exp_backoff_fn(fn, *args):
         return fn(*args)
 
     import random
-    max_retries = 5
-    for n in range(max_retries):
+    # with max_tries = 5, max total time ~= 3.2 sec
+    # with max_tries = 6, max total time ~= 6.5 sec
+    max_tries = 6
```

Comment:
For the sake of expediency I think this is good, but I like Ray's idea of maxing out the delays at 1 second.